lms/envs/common.py

```diff
@@ -1210,20 +1210,21 @@
 
 ############################### PIPELINE #######################################
 
+PIPELINE_ENABLED = True
+
 # Process static files using RequireJS Optimizer
 STATICFILES_STORAGE = 'openedx.core.lib.django_require.staticstorage.OptimizedCachedRequireJsStorage'
 
 # List of finder classes that know how to find static files in various locations.
 # Note: the pipeline finder is included to be able to discover optimized files
 STATICFILES_FINDERS = [
-    'staticfiles.finders.FileSystemFinder',
-    'staticfiles.finders.AppDirectoriesFinder',
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
     'pipeline.finders.PipelineFinder',
 ]
 
-# Don't use compression by default
 PIPELINE_CSS_COMPRESSOR = None
-PIPELINE_JS_COMPRESSOR = None
+PIPELINE_JS_COMPRESSOR = 'pipeline.compressors.uglifyjs.UglifyJSCompressor'
```

Comment:
@cpennington Currently, `PIPELINE_JS_COMPRESSOR` is set to `UglifyJSCompressor` in aws.py and `None` in common. I have just moved this here from aws.py and explicitly disabled the compressor in env files where it is off.

There was a discussion on https://github.com/edx/edx-platform/pull/9059 and https://github.com/edx/edx-platform/pull/9803#discussion_r39856565 about what is the right convention for this. Usually common.py has the settings we would be running in production with and if we want to have a different setting for tests or devstack, we set that in the appropriate places. This change is just to adopt that convention for this setting.